database/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
 
 # Create your models here.
+class Request(models.Model):
+    request_ID = models.IntegerField(max_length=50)
+    user_access_ID= models.CharField(max_length=250)
+    request_time = models.DateTimeField(max_length=100)
+
+class User(models.Model):
+    user_id = models.IntegerField(max_length=50)
+    email = models.EmailField(max_length=100)
+    password = models.CharField(max_length=250)
+    user_type = models.CharField(max_length=20)
+
+class Adviser(models.Model):
```

Comment:
I think this should have a relationship with Section? I think it should be a [ManyToMany](https://docs.djangoproject.com/en/2.2/topics/db/examples/many_to_many/) relationship (even though in the ERD it's only a one-to-many; I left a comment there suggesting that it should be ManyToMany).

Reply:
done